src/components/textarea/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {
+  ChangeEventHandler,
+  FocusEventHandler,
+  forwardRef,
+} from 'react';
+
+import { Textarea as ChakraTextarea } from '@chakra-ui/react';
+
+export interface TextareaProps {
+  name: string;
+  value?: string;
+  placeholder?: string;
+  disabled?: boolean;
+  autoFocus?: boolean;
+  onBlur?: FocusEventHandler<HTMLTextAreaElement>;
+  onFocus?: FocusEventHandler<HTMLTextAreaElement>;
+  onChange?: ChangeEventHandler<HTMLTextAreaElement>;
+  rows?: number;
+  cols?: number;
+  textStyle?: 'body' | 'body-small';
+}
+
+export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ ...props }, ref) => {
+    return <ChakraTextarea {...props} ref={ref} />;
+  },
+);
```

Comment:
I'm wondering - do we gain a lot by restricting props here? Wouldn't re-exporting from chakra with a display name reduce complexity on our side?

Reply:
I would not challenge that during the migrations as it's the agreed approach to expose the minimal needed props for components to enforce the desing system. If we wanna take that discussion i would do that separate form the migration